dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts

```diff
@@ -205,13 +205,17 @@ export class CustomPanelsAdaptor {
       const response = await client.callAsCurrentUser('observability.getObject', {
         objectType: 'savedVisualization',
       });
-      return response.observabilityObjectList.map((visualization: any) => ({
-        id: visualization.objectId,
-        name: visualization.savedVisualization.name,
-        query: visualization.savedVisualization.query,
-        type: visualization.savedVisualization.type,
-        timeField: visualization.savedVisualization.selected_timestamp.name,
-      }));
+      return response.observabilityObjectList
+        .filter((visualization: any) => {
+          return !!!visualization.savedVisualization.application_id;
```

Comment:
Question: Was this done by eslint autocorrection? Do we need 3 '!'?

Reply:
It is the same as ! but it shows that I'm evaluating a truthy/falsy value instead of a boolean.